### PR TITLE
chore: added IC-2023.2 to verified platforms (#193)

### LINKED ISSUE #193: add IC-2023-2 to verification

### PR DESCRIPTION
**Pull request by adietish**
fixes #193

### CODE REVIEW

File: .github/workflows/IJ.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        IJ: [IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2, , IC-2022.3, IC-2023.1]
+        IJ: [IC-2021.1, IC-2021.2, IC-2021.3, IC-2022.1, IC-2022.2, , IC-2022.3, IC-2023.2]
```
Comment (sbouchet) [inline .github/workflows/IJ.yml:17]: why removing 2023.1 ?

**Comment by sbouchet** [comment]
seems like some work needs to be done for next release of IJ...

**Comment by adietish** [comment]
> seems like some work needs to be done for next release of IJ...

yes, there's stuff to fix before this is ready. I noticed this end of last week before devtools week and thus set it to draft.

**Comment by adietish** [comment]
@sbouchet please review. Quite some changes. Many caused by the need to compile using jdk17 when using IC-2023.2.

**Review by sbouchet — APPROVED** [review]
LGTM. 
however but seems not causing any troubles yet, there is a warning in all versions on : 
` /home/runner/work/intellij-common/intellij-common/src/test/java/com/redhat/devtools/intellij/common/actions/editor/RemoveYAMLClutterActionHandlerTest.java uses or overrides a deprecated API.`

**Pull request merged**